enterprise/storage/saas_app_conversation_info_injector.py

```diff
@@ -0,0 +1,350 @@
+"""Enterprise injector for SQLAppConversationInfoService with SAAS filtering."""
```

Comment:
super tiny NIT: i think this file would be better located somewhere outside `storage` subdirectory